elements/lisk-p2p/src/peer_book/base_list.ts

```diff
@@ -12,8 +12,11 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+// eslint-disable-next-line import/no-cycle
 import { ExistingPeerError } from '../errors';
+// eslint-disable-next-line import/no-cycle
 import { P2PEnhancedPeerInfo, P2PPeerInfo } from '../types';
+// eslint-disable-next-line import/no-cycle
```

Comment:
If there are group of similar rules can be clubbed to make. it looks cleaner. if you accept this suggestion, please use the same in other places.
```suggestion
/* eslint-disable import/no-cycle */
import { ExistingPeerError } from '../errors';
import { P2PEnhancedPeerInfo, P2PPeerInfo } from '../types';
import {
	evictPeerRandomlyFromBucket,
	getBucketId,
	PEER_TYPE,
	sanitizeEnhancedPeerInfo,
} from '../utils';
/* eslint-enable */
```

Reply:
For the no-cycle, we want to fix this issue one by one, so I think it make sense not to group them